analyzers/tests/SonarAnalyzer.Test/Rules/SecureRandomSeedsShouldNotBePredictableTest.cs

```diff
@@ -230,8 +276,7 @@ class MyGen : IRandomGenerator
                     public void NextBytes(byte[] bytes, int start, int len) { }
                     public void NextBytes(Span<byte> bytes) { }
                 }
-
                 """)
-            .VerifyNoIssues();
+            .Verify();
```

Comment:
I added a failing case as a baseline, because `VerifyNoIssues` might mean the rule is not running.